organizacja-hacknightu.md

```diff
@@ -0,0 +1,39 @@
+# Hacknight
+
+1. Spotykamy się punktualnie.
+1. Na początku spotkania ustalamy listę tematów/rzeczy, którymi mamy się zająć, określamy też czas, jaki chcemy poświęcić na dane zagadnienie.
+1. Jedna osoba pilnuje czasów i tematów, może też prowadzi spotkanie?
+1. Jeśli mamy tematy, w które nie wszyscy muszą być włączeni, pracujemy w podgrupach, najlepiej w różnych miejscach, tak, by sobie nie przeszkadzać.
+1. Pod koniec spotkania spotykamy się i mówimy, co ustaliliśmy, co zrobiliśmy i czym mamy się zająć na kolejnym spotkaniu.
+1. Może raz w miesiącu (?) planowanie działań i sprawdzanie, w którym punkcie jesteśmy?
+1. Onboarding zawsze odbywa się równolegle, w osobnym pomieszczeniu.
+
+# Onboarding
+
+Najlepiej, gdyby to robiły dwie osoby, jedna od spraw technicznych, druga od ogólnych / onboarding będziemy opracowywać, najpewniej, w osobnym dokumencie | Wojtek /
```

Comment:
na to najlepiej od razu założyć osobne issue (no chyba, że @pannami tak nie uważa)